File: program126.java
import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        char ch = '\0';
        ch = 'A';
        for(iCnt = 1;iCnt <= iNo;iCnt++)
        {
            System.out.print(ch+" \t");
            ch++;
        }
    }
}
class program126
{
    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the values");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue);
    }
}

File: program126.c
#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;
    char cCnt = '\0';
    for(cCnt = 1;cCnt<=iNo;cCnt++)
    {
        printf("%c \t",cCnt);
    }

}

int main()
{
    int iValue = 0;

    printf("Enter the number of elements");
    scanf("%d",&iValue);
    Pattern(iValue);
    return 0;
}